src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx

```diff
@@ -137,26 +131,15 @@ function NodeDetailComponent({
                     )
                 }
 
-                if (Array.isArray(result.manifest.spec.ephemeralContainers)) {
-                    const ephemeralContainerStatusMap = new Map<string, string[]>()
-                    result.manifest.spec.ephemeralContainers.forEach((con) => {
-                        ephemeralContainerStatusMap.set(con.name, con.command as string[])
-                    })
+                if (Array.isArray(result.ephemeralContainers)) {
```

Comment:
Check if this still needs to be under the if at line 113 or the condition needs to be updated

Reply:
done